### ISSUE TITLE
loss of GAN

### ISSUE THREAD

**Issue by viridyu** [junyanz/pytorch-CycleGAN-and-pix2pix#164]

**Comment by viridyu** [comment]
Hi 
I have trained cycleGan on my own dataset. During the training, I found that the GAN loss of Generator A cannot be convergent and becomes larger. At the same time, the GAN loss of Generator B, Discriminator A and B all go convergent. Have anyone met this problem before? Thank you.

**Comment by junyanz** [comment]
It is quite normal. It is fine as long as the losses do not explode. We are using LSGAN and there is no guarantee that the GAN loss will converge. You can get a convergent loss if you are using [WGAN](https://arxiv.org/abs/1701.07875
)(or [WGAN-GP](https://arxiv.org/abs/1704.00028)).
